### PR TITLE
Small svg tweak

### PR DESCRIPTION
**Pull request by juliendonck**
Fixed the warning svg!
Screenshot 2020-03-24 at 10 22 53

First PR 🔥

### CODE REVIEW

File: ui/DesignSystem/Primitive/Icon/TxState.svelte [hunk 1]
```diff
@@ -93,18 +93,26 @@
...
+      10.8197 17.514 10.9605 17.5 11.1L17 16.1Z"
+      fill={color} />
+    <path
+      opacity="0.3"
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M16 4C9.37258 4 4 9.37258 4 16C4 22.6274 9.37258 28 16 28C22.6274 28 28
+      22.6274 28 16C28 9.37258 22.6274 4 16 4ZM0 16C0 7.16344 7.16344 0 16
+      0C24.8366 0 32 7.16344 32 16C32 24.8366 24.8366 32 16 32C7.16344 32 0
+      24.8366 0 16Z"
```
Comment (MeBrei) [inline ui/DesignSystem/Primitive/Icon/TxState.svelte]: Is this the circle around the exclamation mark? It is already implemented above (l.74)

Comment (juliendonck) [reply ui/DesignSystem/Primitive/Icon/TxState.svelte]: No, it's the little circle at the bottom of the exclamation mark.

Comment (MeBrei) [reply ui/DesignSystem/Primitive/Icon/TxState.svelte]: If you change the colors of the 2 paths to something other than red (each a different color) you can see that the first path is the full exclamation mark (little circle + vertical line) and the other one is an additional (opaque) circle ie the progress bar circle that is already implemented via `circle` above.

Comment (juliendonck) [reply ui/DesignSystem/Primitive/Icon/TxState.svelte]: Lol my bad, I just assumed. Will push an update...

**Pull request merged**